runner/README.md

```diff
@@ -0,0 +1,41 @@
+package runner is structs and interfaces to run Scoot processes.
+
+## Summary ##
+* Invoker runs one Command using Execer (and snapshot.Filer, etc.)
+* Controller controls when to Run something (using an Invoker)
+* Statuses queries Process history
+  * StatusesSync doesn't offer waiting, so requires client polling
+  * LegacyStatuses is the old Read API
+  * StatusWriter writes Process status (called by Controller)
+
+## Invoker (invoker.go) ##
+Invoker runs a Scoot command. This involves:
+* setup (check out a snapshot)
+* post-processing (save output)
+* babysitting (implement timeout and abort)
+* updates while running
+
+Conceptually, it lowers a Scoot Command from a high-level abstraction (Snapshots, etc.) to an Execer command (about directories that are paths in the local filesystem).
+
+## Controller (controller.go) ##
+Controller controls the Runs, starting a new one (Run()) or ending one (Abort()). It implements:
+* tenancy (run one at a time? Or multiple? Fixed, or based on system utilization?)
+* queueing (should we accept work we can't start right now)
+
+When a Controller wants to start a Run of a Command, it will use an Invoker.
+
+## Statuses (statuses.go) ##
+Statuses allows querying for Process Status.
```

Comment:
we should describe what a 'Process Status' ... if its the status of a 'Run of a Command' why don't we call it Run Status or Command Status - where did the word 'Process' come from?